sdk/batch/batch/README.md

```diff
@@ -19,91 +19,50 @@ npm install @azure/batch
-import { BatchServiceClient, BatchServiceModels, BatchServiceMappers } from "@azure/batch";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new BatchServiceClient(creds, subscriptionId);
-  const maxResults = 1;
-  const timeout = 1;
-  const clientRequestId = ec7b1657-199d-4d8a-bbb2-89a11a42e02a;
-  const returnClientRequestId = true;
-  const ocpDate = new Date().toUTCString();
-  client.application.list(maxResults, timeout, clientRequestId, returnClientRequestId, ocpDate).then((result) => {
-    console.log("The result is:");
-    console.log(result);
-  });
-}).catch((err) => {
-  console.error(err);
-});
-```
-
-#### browser - Authentication, client creation and list application as an example written in JavaScript.
-
-##### Install @azure/ms-rest-browserauth
-
-```
-npm install @azure/ms-rest-browserauth
-```
-
-##### Sample code
-
-See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to Azure in the browser.
-
-- index.html
-```html
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <title>@azure/batch sample</title>
-    <script src="node_modules/@azure/ms-rest-js/dist/msRest.browser.js"></script>
-    <script src="node_modules/@azure/ms-rest-azure-js/dist/msRestAzure.js"></script>
-    <script src="node_modules/@azure/ms-rest-browserauth/dist/msAuth.js"></script>
-    <script src="node_modules/@azure/batch/dist/batch.js"></script>
-    <script type="text/javascript">
-      const subscriptionId = "<Subscription_Id>";
-      const authManager = new msAuth.AuthManager({
-        clientId: "<client id for your Azure AD app>",
-        tenant: "<optional tenant for your organization>"
-      });
-      authManager.finalizeLogin().then((res) => {
-        if (!res.isLoggedIn) {
-          // may cause redirects
-          authManager.login();
-        }
-        const client = new Azure.Batch.BatchServiceClient(res.creds, subscriptionId);
-        const maxResults = 1;
-        const timeout = 1;
-        const clientRequestId = ec7b1657-199d-4d8a-bbb2-89a11a42e02a;
-        const returnClientRequestId = true;
-        const ocpDate = new Date().toUTCString();
-        client.application.list(maxResults, timeout, clientRequestId, returnClientRequestId, ocpDate).then((result) => {
-          console.log("The result is:");
-          console.log(result);
-        }).catch((err) => {
-          console.log("An error occurred:");
-          console.error(err);
-        });
-      });
-    </script>
-  </head>
-  <body></body>
-</html>
+```typescript
+import { BatchServiceClient, BatchServiceModels, BatchSharedKeyCredentials } from "@azure/batch";
+
+const batchAccountName = process.env["AZURE_BATCH_ACCOUNT_NAME"] || "";
+const batchAccountKey = process.env["AZURE_BATCH_ACCOUNT_KEY"] || "";
+const batchEndpoint = process.env["AZURE_BATCH_ENDPOINT"] || "";
+
+const creds = new BatchSharedKeyCredentials(batchAccountName, batchAccountKey);
```

Comment:
it might be better to add documentation on authenticating via Managed identity as well.

`const creds =  await msRestNodeAuth.loginWithVmMSI({resource: 'https://batch.core.windows.net/'})`